RecommendationEngineClient/RecommendationEngineClient/10 Common/DTO/ChefDTO.cs

```diff
@@ -0,0 +1,33 @@
+﻿using RecommendationEngineClient.Common.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RecommendationEngineClient._10_Common.DTO
+{
+
+    public class MenuItem
+    { 
+      public DateTime CurrentDate { get; set; }
+      public List<int> MenuItemsIds { get; set; }
+    
+    }
+
```

Comment:
formatting. unnecessary multiple line breaks. see if you can use any code formatting extension like prettify

Reply:
@chetanj257 As per my knowledge there are no extensions available such as prettify we have to do it manually in Visual Studio

Reply:
@chetanj257 I have formatted all the files manually this issue is resolved from my side